src/helper.jsx

```diff
@@ -0,0 +1,4 @@
+
+export const getFindId = (tabs, selectedTabId) => (
```

Comment:
One verb is enough for the name of this function
```suggestion
export const getTabById = (tabs, selectedTabId) => (
```
```suggestion
export const findTabById = (tabs, selectedTabId) => (
```